### DISCUSSION TITLE
What is the difference between all the ESLint Nextjs plugins

### DISCUSSION THREAD

**Discussion by nattstack** [vercel/next.js#74314] [Help]
### Summary

Hello, I was reading [ESLint Plugin Docs](https://nextjs.org/docs/app/api-reference/config/eslint), and I am a confused I should set up my ESLint. If I want all of the Nextjs rules, how should I set up without overlap.

```js
const eslintConfig = [
  ...compat.config({ extends: ["next", "next/typescript", "prettier"] })
]

const eslintConfig = [
  ...compat.config({ extends: ["next", "next/core-web-vitals", "next/typescript", "prettier"] })
]

const eslintConfig = [
  ...compat.config({ extends: ["next", "prettier"] })
]
```

**Comment by icyJoseph** [comment]
I think there is a pretty good answer here: https://github.com/vercel/next.js/discussions/58714#discussioncomment-8324062

**Comment by barnejs** [comment]
*eslint-config-next/core-web-vitals* is the strictest next config and
includes everything.
